packages/dds/tree/src/feature-libraries/sequence-field/compose.ts

```diff
@@ -324,11 +347,6 @@ function composeMarksIgnoreChild(
 	}
 
 	if (!markHasCellEffect(baseMark) && !markHasCellEffect(newMark)) {
-		if (isNoopMark(baseMark)) {
```

Comment:
What was the motivation for moving this code instead of putting the rename logic in here?

Reply:
The block of logic for handling rename marks gets to be more cohesive (as opposed to adding logic inside multiple existing branches) if it appears earlier. It also gets to be simpler if the noop marks are handled before it. Since handling the noop mark is itself simple and self-contained it seemed reasonable to order things that way.